src/test/librados/crimson_utils.h

```diff
@@ -0,0 +1,13 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <cstdlib>
+
+bool is_crimson_cluster();
```

Comment:
not sure if this tiny function deserves a .cc file, how about
```c++
static inline bool is_crimson_cluster() {
  return getenv("CRIMSON_COMPAT") != nullptr;
}
```

also, i'd suggest drop the parenthesis around the expression.

Reply:
done.